modules/clearUi.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable*/
+const input = document.querySelector('.input');
+
+const clear = () => {
+  input.value = '';
+};
+
```

Comment:
- [ ] [REQUIRED] Welldone so far but I notice a few things:
    - Please ensure that the page reloads and updates after any interaction applied to your program. Eg. the checkbox does not update in the local storage until I manually refresh. 
    - Secondly, after completed is marked true, if I reload the page, the UI does reflect the completed status initially applied. 

Please check your code again and ensure that interactions are stored and loaded correctly on reload. in other words, the updated array of items is stored in local storage, so the user gets the correct list values after the page reloads.